examples/expressions/main.rs

```diff
@@ -0,0 +1,84 @@
+mod metadata;
+mod note;
+
+use gtk::gio;
+use gtk::glib;
+use gtk::prelude::*;
+
+use self::metadata::Metadata;
+use self::note::Note;
+
+fn main() {
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.expressions"),
+        Default::default(),
+    );
+
+    application.connect_activate(build_ui);
+    application.run();
+}
+
+fn build_ui(app: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(app);
+    window.set_title(Some("Expressions"));
+    window.set_default_size(300, 500);
+
+    let factory = gtk::SignalListItemFactory::new();
+    factory.connect_setup(|_, list_item| {
+        let hbox = gtk::Box::new(gtk::Orientation::Horizontal, 24);
+
+        let title_label = gtk::Label::new(None);
+        hbox.append(&title_label);
+
+        let last_modified_label = gtk::Label::new(None);
+        hbox.append(&last_modified_label);
+
+        // Instead of binding properties and manually unbinding them, you can create expressions.
+        // The value can be obtained even if it is several steps away.
+        let metadata_expression = list_item
+            .property_expression("item")
+            .chain_property(Note::static_type(), "metadata");
+
+        metadata_expression
+            .chain_property(Metadata::static_type(), "title")
+            .chain_closure(|args| {
+                let title: String = args[1].get().unwrap();
+                format!("Title: {}", title)
+            })
+            .bind(&title_label, "label", None::<&gtk::Widget>);
```

Comment:
```suggestion
            .bind(&title_label, "label", gtk::NONE_WIDGET);
```

We chould use the const here.
No strong preference though.

Reply:
I think that may be replaced now with gtk::Widget::NONE, so I think replace it with this const?

Reply:
Good catch.

It should be `gtk::Widget::NONE`
I've updated the suggestion.

Reply:
thanks

Reply:
thanks